spec/presenters/coronavirus/sub_section_json_presenter_spec.rb

```diff
@@ -12,6 +12,7 @@
 
       expected_output = {
         title: sub_section.title,
+        sub_heading: nil,
```

Comment:
Good to know it handles nil! but can use the factory like you have in the other tests: `sub_section.sub_heading`